src/Resources/config/services.yml

```diff
@@ -21,3 +21,13 @@ services:
         arguments: ['%okvpn_fixture.table%']
         tags:
             - { name: doctrine.event_listener, event: loadClassMetadata }
+
+    okvpn_fixture.command.load_data_fixtures:
+        class: Okvpn\Bundle\FixtureBundle\Command\LoadDataFixturesCommand
+        arguments:
+            - '@doctrine'
+            - '@okvpn_fixture.data.loader'
+            - '@okvpn_fixture.data.executor'
+            - '@parameter_bag'
```

Comment:
Will not works with SF < 4.1. A new `parameter_bag` service was added only since 4.1